pyro/poutine/guide.py

```diff
@@ -112,7 +112,7 @@ def get_posterior(
         """
         raise NotImplementedError
 
-    def upstream_value(self, name: str) -> torch.Tensor:
+    def upstream_value(self, name: str):
```

Comment:
Temporarily removed the return type (changed to Any) so that mypy wouldn't complain.